lib/rex/proto/ldap.rb

```diff
@@ -5,7 +5,16 @@
 # TODO: write a real LDAP client in Rex and migrate all consumers
 class Net::LDAP::Connection # :nodoc:
   module SynchronousRead
-    def read(length = nil, opts = {})
+    #
+    # Read `length` bytes of data from the LDAP connection socket and
+    #  return this data as a string.
```

Comment:
```suggestion
    # return this data as a string.
```

![image](https://user-images.githubusercontent.com/58950994/210890102-6e6adda2-7b19-4d12-8391-b3e3a8e24447.png)

Reply:
Woops thought I fixed all these issues. Apparently the description doesn't have to be indented for long lines but anything other than the description does which leads to some confusion 😖

Reply:
Fixed in https://github.com/rapid7/metasploit-framework/pull/17427/commits/e03fd42a299a18ec7ed64ad908ddb9af9154ae39